src/System.Runtime.CompilerServices.Unsafe/src/Configurations.props

```diff
@@ -2,6 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
+      netcoreapp;
```

Comment:
You'll need to split this between BuildConfigurations and PackageConfigurations.
```xml
    <PackageConfigurations>
      netstandard1.0;
      netstandard;
    </PackageConfigurations>
    <BuildConfigurations>
      $(PackageConfigurations);
      netcoreapp;
    </BuildConfigurations>
```